pkg/test/oci-layout/oci_layout_test.go

```diff
@@ -431,11 +438,95 @@ func TestExtractImageDetails(t *testing.T) {
 			panic(err)
 		}
 
-		olu := NewBaseOciLayoutUtils(storeController, testLogger)
+		olu := ocilayout.NewBaseOciLayoutUtils(storeController, testLogger)
 		resDigest, resManifest, resIspecImage, resErr := olu.ExtractImageDetails("zot-test", "latest", testLogger)
 		So(resErr, ShouldEqual, zerr.ErrBlobNotFound)
 		So(string(resDigest), ShouldEqual, "")
 		So(resManifest, ShouldBeNil)
 		So(resIspecImage, ShouldBeNil)
 	})
 }
+
+func getTags() ([]cvemodel.TagInfo, []cvemodel.TagInfo) {
+	tags := make([]cvemodel.TagInfo, 0)
+
+	firstTag := cvemodel.TagInfo{
+		Name: "1.0.0",
+		Descriptor: cvemodel.Descriptor{
+			Digest:    "sha256:eca04f027f414362596f2632746d8a178362170b9ac9af772011fedcc3877ebb",
+			MediaType: ispec.MediaTypeImageManifest,
+		},
+		Timestamp: time.Now(),
+	}
+	secondTag := cvemodel.TagInfo{
+		Name: "1.0.1",
+		Descriptor: cvemodel.Descriptor{
+			Digest:    "sha256:eca04f027f414362596f2632746d8a179362170b9ac9af772011fedcc3877ebb",
+			MediaType: ispec.MediaTypeImageManifest,
+		},
+		Timestamp: time.Now(),
+	}
+	thirdTag := cvemodel.TagInfo{
+		Name: "1.0.2",
+		Descriptor: cvemodel.Descriptor{
+			Digest:    "sha256:eca04f027f414362596f2632746d8a170362170b9ac9af772011fedcc3877ebb",
+			MediaType: ispec.MediaTypeImageManifest,
+		},
+		Timestamp: time.Now(),
+	}
+	fourthTag := cvemodel.TagInfo{
+		Name: "1.0.3",
+		Descriptor: cvemodel.Descriptor{
+			Digest:    "sha256:eca04f027f414362596f2632746d8a171362170b9ac9af772011fedcc3877ebb",
+			MediaType: ispec.MediaTypeImageManifest,
+		},
+		Timestamp: time.Now(),
+	}
+
+	tags = append(tags, firstTag, secondTag, thirdTag, fourthTag)
+
+	vulnerableTags := make([]cvemodel.TagInfo, 0)
+	vulnerableTags = append(vulnerableTags, secondTag)
+
+	return tags, vulnerableTags
+}
+
+func TestTagsInfo(t *testing.T) {
+	Convey("Test tags info", t, func() {
+		allTags, vulnerableTags := getTags()
+
+		latestTag := ocilayout.GetLatestTag(allTags)
+		So(latestTag.Name, ShouldEqual, "1.0.3")
+
+		fixedTags := cveinfo.GetFixedTags(allTags, vulnerableTags)
+		So(len(fixedTags), ShouldEqual, 2)
+
+		fixedTags = cveinfo.GetFixedTags(allTags, append(vulnerableTags, cvemodel.TagInfo{
+			Name:       "taginfo",
+			Descriptor: cvemodel.Descriptor{},
+			Timestamp:  time.Date(2000, time.July, 20, 10, 10, 10, 10, time.UTC),
+		}))
+		So(len(fixedTags), ShouldEqual, 3)
+
+		log := log.NewLogger("debug", "")
```

Comment:
Code starting here up until `repo, digest := common.GetImageDirAndDigest("image")` should not be needed anymore. The function it tested was removed.